crates/server/src/admin/handlers/context.rs

```diff
@@ -13,8 +14,10 @@ use crate::admin::storage::client_keys::get_context_client_key;
 use crate::admin::utils::context::{create_context, join_context};
 
 #[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
 pub struct ContextObject {
     context: calimero_primitives::context::Context,
+    context_identities: Vec<String>,
```

Comment:
will we always want all identities for the context when we're getting context metadata?

I think maybe this should be a different endpoint, wdyt?

Reply:
I think it is relevant context metadata the same as app_id or last_tx_hash. 

Use case for this is to select with which identity you want to join selected context or in context details show context config data. 

Also, the payload should be small, as user won't have too much identities for a single context. I don't see such use case.